src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,27 @@
+@mixin transitionDuration {
+  transition: all 0.3s;
+}
```

Comment:
Do not use this mixin, `all` transition-property it's hard for browser, use transition where require with proprties you change on hover. For example
`transition: color 0.3s, trasform 0.3s`